.github/workflows/broken_links_scheduled.yml

```
@@ -0,0 +1,68 @@
+name: Check for broken links
```

```suggestion
name: Check for internal and external broken links
```

And maybe rename the other one too